recipes/corrade/all/conanfile.py

```diff
@@ -25,16 +24,18 @@ class CorradeConan(ConanFile):
         "with_pluginmanager": [True, False],
         "with_testsuite": [True, False],
         "with_utility": [True, False],
+        "with_rc": [True, False],
     }
     default_options = {
         "shared": False,
         "fPIC": True,
-        "build_deprecated": True,
+        "build_deprecated": False,
```

Comment:
```suggestion
```

Build deprecated is invalid, you can remove this option.

Reply:
Nah, [it's still a valid option](https://github.com/mosra/corrade/blob/61d1b58cbcb159837bca506b5336a810da67f0a7/CMakeLists.txt#L62-L65)...